src/components/ImageGenerator.tsx

```
@@ -0,0 +1,159 @@
+                outputFormat: "WEBP",
+                CFGScale: 7,
+                steps: 20
+              }
+            ])
+          });
+
+          const data = await response.json();
+          
+          if (data.data && data.data.length > 0) {
+            const imageResult = data.data.find((item: any) => item.taskType === "imageInference");
+            if (imageResult && imageResult.imageURL) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)